index.html

```diff
@@ -8,61 +8,98 @@
     <title>Company Site</title>
     <META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW" />
   </head>
+
   <body>
-    <nav>
-      <a class="logo" href="#">
+    <header class="header">
```

Comment:
I typically use the header tag for the whole header element, and then the nav tag for the actual navigation (links, dropdowns, buttons, etc.) 

This is good practice for accessibility too, gotta get them semantic tags in!!!